easybuild/easyconfigs/t/Tk/Tk-8.6.4-intel-2016a-no-X11.eb

```diff
@@ -20,6 +20,7 @@ dependencies = [
     ('zlib', '1.2.8'),
 ]
 
+# To be clear: this will still require the X11 to be present.
```

Comment:
also, this is totally unrelated to this PR (this particular Tk easyconfig isn't used here, it's even a different toolchain)

let's not start adding comments left and right for stuff we already have open issues for